app/src/main/java/com/babylon/wallet/android/presentation/transaction/composables/TransactionExpirationInfo.kt

```diff
@@ -83,20 +83,28 @@ fun PreAuthorizationFeesHint(
     proposingDApp: TransactionReviewViewModel.State.ProposingDApp
 ) {
     Row(
-        modifier = modifier.background(
-            color = RadixTheme.colors.gray5,
-            shape = RadixTheme.shapes.roundedRectMedium
-        ).clickable {
-            onClick()
-        }.padding(
-            vertical = RadixTheme.dimensions.paddingDefault,
-            horizontal = RadixTheme.dimensions.paddingSemiLarge
-        ),
+        modifier = modifier
+            .background(
+                color = RadixTheme.colors.gray5,
+                shape = RadixTheme.shapes.roundedRectMedium
+            )
+            .clickable {
+                onClick()
+            }
+            .padding(
+                vertical = RadixTheme.dimensions.paddingDefault,
+                horizontal = RadixTheme.dimensions.paddingSemiLarge
+            ),
         verticalAlignment = Alignment.CenterVertically
     ) {
         Column(modifier = Modifier.weight(1f)) {
             Text(
-                text = stringResource(R.string.preAuthorizationReview_fees_title, proposingDApp.name.orEmpty()),
+                text = stringResource(
+                    R.string.preAuthorizationReview_fees_title,
+                    proposingDApp.name.orEmpty().ifEmpty {
```

Comment:
@sergiupuhalschi-rdx We also have this:`@Composable() DApp?.displayName()` function which you can reuse.